#! /bin/sh /usr/share/dpatch/dpatch-run
## 40_fix_paths.dpatch by Henrique de Moraes Holschuh <hmh@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix references to paths that are different in Debian

@DPATCH@
diff -urNad amavisd-new~/README_FILES/README.chroot amavisd-new/README_FILES/README.chroot
--- amavisd-new~/README_FILES/README.chroot	2006-01-14 02:02:09.000000000 +0100
+++ amavisd-new/README_FILES/README.chroot	2007-09-17 22:33:50.000000000 +0200
@@ -23,6 +23,9 @@
 If you have Postfix, check its chroot setup script for further hints:
 postfix-xxx/examples/chroot-setup/<YOUR-OS> and BASIC_CONFIGURATION_README.
 
+WARNING: Debian uses different paths, notably /var/lib/amavisd-new.  Be
+sure to adjust the directions in this document accordingly.
+
 
 exit   # This is NOT an automatic script!!!
        # Don't execute commands without knowing what they will do!!!
diff -urNad amavisd-new~/README_FILES/README.old.scanners amavisd-new/README_FILES/README.old.scanners
--- amavisd-new~/README_FILES/README.old.scanners	2005-04-25 01:16:20.000000000 +0200
+++ amavisd-new/README_FILES/README.old.scanners	2007-09-17 22:33:50.000000000 +0200
@@ -89,7 +89,7 @@
 otherwise your logfiles don't show which file(s) is/are infected.
 
 NOTE: AvpDaemon must be running as a daemon, so it should be started at
-boot time via an init script (or whatver) as <path>/AvpDaemon -* /var/amavis
+boot time via an init script (or whatver) as <path>/AvpDaemon -* /var/lib/amavis
 
 3.1.2 Kaspersky Anti-Virus
 AvpLinx fills the log with a lot of trash because of a simple progress
@@ -799,7 +799,7 @@
 
 # Now restart AVP daemon to load updated virus library
 system("/usr/local/AvpLinux/AvpDaemon -k");
-system("/usr/local/AvpLinux/AvpDaemon -* /var/amavis");
+system("/usr/local/AvpLinux/AvpDaemon -* /var/lib/amavis");
 
 # End of perl script
 
diff -urNad amavisd-new~/README_FILES/README.performance amavisd-new/README_FILES/README.performance
--- amavisd-new~/README_FILES/README.performance	2005-08-12 16:15:59.000000000 +0200
+++ amavisd-new/README_FILES/README.performance	2007-09-17 22:33:50.000000000 +0200
@@ -16,7 +16,7 @@
 Hopefully hardware matches expectations,
 fast disks and enough memory are paramount.
 
-You may want to put Postfix spool on different disk than /var/amavis,
+You may want to put Postfix spool on different disk than /var/lib/amavis,
 where amavisd does mail unpacking.
 
 | is there any suggested configuration for this
diff -urNad amavisd-new~/README_FILES/README.sendmail amavisd-new/README_FILES/README.sendmail
--- amavisd-new~/README_FILES/README.sendmail	2005-04-25 01:16:34.000000000 +0200
+++ amavisd-new/README_FILES/README.sendmail	2007-09-17 22:33:50.000000000 +0200
@@ -212,7 +212,7 @@
 
 /var/spool/mqueue and /var/spool/mqamavis is owned by amavis.
 
-NOTE: as amavis is run as user amavis, /var/virusmails must be owned
+NOTE: as amavis is run as user amavis, /var/lib/amavis/virusmails must be owned
 by amavis and you have to specify a location for the AMaViS logfile
 that is writable by user amavis, if writing to a log file directly
 (not via syslog).
@@ -291,9 +291,9 @@
 # i       Deliver interactively (synchronously)
 
 
-NOTE: as amavis is run as amavis /var/virusmails must be owned
+NOTE: as amavis is run as amavis /var/lib/amavis/virusmails must be owned
 by amavis and you have to specify a another location for the AMaViS
-logfile (normally /var/amavis/amavis.log) to which amavis has
+logfile (normally /var/lib/amavis/amavis.log) to which amavis has
 write access to.
 
 NOTE: As sendmail will perform most tasks as user amavis now, it may
diff -urNad amavisd-new~/README_FILES/README.sendmail-dual amavisd-new/README_FILES/README.sendmail-dual
--- amavisd-new~/README_FILES/README.sendmail-dual	2006-09-15 22:36:47.000000000 +0200
+++ amavisd-new/README_FILES/README.sendmail-dual	2007-09-17 22:33:50.000000000 +0200
@@ -427,8 +427,8 @@
 
 - Mail handling is I/O-intensive. For better performance one may place
   the two mail queue areas (/var/spool/mqueue and /var/spool/mqueue-rx),
-  and the /var/amavis work directory ($TEMPBASE) on three separate disks.
-  The /var/amavis/tmp may be a tmpfs or a RAMdisk or delayed-sync fs.
+  and the /var/lib/amavis work directory ($TEMPBASE) on three separate disks.
+  The /var/lib/amavis/tmp may be a tmpfs or a RAMdisk or delayed-sync fs.
 
 - One of the important arguments for choosing the dual-MTA setup is to be
   able to keep the number of content filtering processes under control,
diff -urNad amavisd-new~/amavisd-agent amavisd-new/amavisd-agent
--- amavisd-new~/amavisd-agent	2007-05-22 14:53:55.000000000 +0200
+++ amavisd-new/amavisd-agent	2007-09-17 22:40:16.000000000 +0200
@@ -146,7 +146,7 @@
       { die "Usage: $0 [ -w <wait-interval> ]\n" }
   }
   my($env) = BerkeleyDB::Env->new(
-    '-Home'=>'/var/amavis/db', '-Flags'=> DB_INIT_CDB | DB_INIT_MPOOL);
+    '-Home'=>'/var/lib/amavis/db', '-Flags'=> DB_INIT_CDB | DB_INIT_MPOOL);
   defined $env or die "BDB no env: $BerkeleyDB::Error $!";
   my($db) = BerkeleyDB::Hash->new(
     '-Filename'=>'snmp.db', '-Flags'=>DB_RDONLY, '-Env'=>$env );
diff -urNad amavisd-new~/amavisd-nanny amavisd-new/amavisd-nanny
--- amavisd-new~/amavisd-nanny	2007-03-29 12:26:48.000000000 +0200
+++ amavisd-new/amavisd-nanny	2007-09-17 22:47:11.000000000 +0200
@@ -56,7 +56,7 @@
 my($activettl) = 10*60; # stuck active children are sent a SIGTERM
                         #   after this many seconds
 
-my($db_home) = '/var/amavis/db';  # DB databases directory
+my($db_home) = '/var/lib/amavis/db';  # DB databases directory
 my($dbfile)  = 'nanny.db';
 my($wakeuptime) = 2;  # sleep time in seconds, may be fractional
 
diff -urNad amavisd-new~/amavisd-release amavisd-new/amavisd-release
--- amavisd-new~/amavisd-release	2007-06-11 14:51:09.000000000 +0200
+++ amavisd-new/amavisd-release	2007-09-17 22:33:50.000000000 +0200
@@ -79,7 +79,7 @@
 
   $log_level = 1;
 # $socketname = '127.0.0.1:9998';
-  $socketname = '/var/amavis/amavisd.sock';
+  $socketname = '/var/run/amavis/amavisd.sock';
 
 sub sanitize_str {
   my($str, $keep_eol) = @_;
